README.md

```diff
@@ -102,6 +102,24 @@ Your project will be compiled ahead of time (AOT), and then the resulting bundle
 - UMD modules result in code that cannot be properly optimized. For best results, prefer ES6 modules whenever possible. This includes third-party dependencies: if one is published in both UMD and ES6 modules, go with the ES6 modules version.
 - During a production build, CommonJs modules will be automatically converted to ES6 modules. This means you can use them and/or require dependencies that use them without any issues.
 
+# Internationalization
+
+Put `i18n` attribute to your html tag to mark it for translation, more information here: https://angular.io/docs/ts/latest/cookbook/i18n.html
+
+## Create a translation source file
+
+```bash
+# Your translation file will be generated here `dist/locale`
+$ npm run i18n
+```
+
+## Production build with your language
+
+```bash
+# Build prod app with the language file `dist/locale/messages.en.xlf`
```

Comment:
Sorry, I didn't explain exactly.

I meant to have the generated files be in `dist` for example, but have the translated files in `src`.
The translation process might be something like:
1. Developer adds `i18n` to code
2. Generate `dist/locale/messages.xlf`
3. Give `messages.xlf` to translators
4. Receive `messages.en.xlf`, `messages.ja.xlf`, ... from translators
5. Check those in version control in `src/assests/locale` for example

Does that sound reasonable to you?